README.md

```diff
@@ -1,2 +1,173 @@
-# Hello-World
-A Hello world Project with Basic HTML and CSS
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello-World](#-hello-world)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Hello-World<a name="Creating Microverse Hello-worl project "></a>
+
+
+**Hello World** is a project build with the knowledge of Html and CSS that has a header.
+
+## 🛠 Built With <a name="HTML&CSS"></a>
+
+### Tech Stack <a name="HTML,CSS and LINTERS"></a>
+
+<details>
+### Key Features <a name="key-features"></a>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https:w3schools.com">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https:w3schools.com">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>LINTERS</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config">Linters</a></li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [NOW AVAILABLE AT THE MOMENT]
+
+## 💻 Getting Started <a name="https://github.com/shurichmond/Hello-World"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
```

Comment:
- [ ] As stated earlier, you can also customize this section to tell future users how to get and run your project. 👋 🦅  here is an example;
-  ### Prerequisites

In order to run this project you need:

```sh
 Node installed on your pc
```

### Setup

Clone this repository to your desired folder:

```sh
git clone https://github.com/shurichmond/Hello-World.git
```

### Install

Install this project with:

```
  cd Hello-World
  npm install
```

Start project with live server